### PR TITLE
Docs - UNR-2325 - Add missing simplayer clients build step

### PR DESCRIPTION
**Pull request by ernopp**
tested with improabdoc

### CODE REVIEW

File: SpatialGDK/Documentation/content/cloud-deployment-workflow.md [hunk 1]
```diff
@@ -71,13 +71,16 @@ You can launch a cloud deployment using the Unreal Editor or the SpatialOS CLI.
...
 6. (Optional) If needed, change the **Region**.
-7. (Optional) Create an additional deployment with [simulated players]({{urlRoot}}/content/simulated-players) that connect to your main game deployment. Simulated players are game clients running in the cloud, mimicking real players of your game from a connection flow and server-worker load perspective. This means they’re useful for scale testing. 
-
-    To create an additional deployment with simulated players, in the **Simulated Players** section:
+7. (Optional) Create an additional deployment with [simulated players]({{urlRoot}}/content/simulated-players) that connect to your main game deployment. Simulated players are game clients running in the cloud, mimicking real players of your game from a connection flow and server-worker load perspective. This means they’re useful for scale testing. </br></br>
+    Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
+    ```
+    BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject
```
Comment (improbable-valentyn) [inline SpatialGDK/Documentation/content/cloud-deployment-workflow.md]: We should probably include both the engine plugin and project plugin alternatives, similar to what we do above for server and client worker assemblies.

File: SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md [hunk 2]
```diff
@@ -192,7 +192,11 @@ You can find out more in the SpatialOS Worker SDK documentation: [launch confgur
   This is: `C:/...<filepath>.../UnrealGDKExampleProject/spatial/snapshots/default.snapshot`. </br></br>
   * **Region**: The real-world geographical location that you want your cloud deployment hosted in. </br>You can change this by selecting a different region from the drop-down list. You might prefer the region you are in.</br></br>
 1. You can also choose to add simulated players via the dialogue box or skip this and move on to item _4._</br></br>
-  To do this, you launch a simulated players deployment with your your game deployment via the **Simulated Players** section of the Cloud Deployment dialog box:</br></br>
+  Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
+  ```
+  BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject
```
Comment (improbable-valentyn) [inline SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md]: Similarly here, we should point out both the engine plugin and project plugin alternatives.

File: SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-cloud.md [hunk 3]
```diff
@@ -193,7 +191,11 @@ You can find out more in the SpatialOS Worker SDK documentation: [launch confgur
   This is: `C:/...<filepath>.../<ProjectRoot>/spatial/snapshots/default.snapshot`. </br></br>
   * **Region**: The real-world geographical location that you want your cloud deployment hosted in. </br>You can change this by selecting a different region from the drop-down list. You might prefer the region you are in.</br></br>
 1. You can also choose to add simulated players via the dialogue box or skip this and move on to item _4._</br></br>
-  To do this, you launch a simulated players deployment with your your game deployment via the **Simulated Players** section of the Cloud Deployment dialog box:</br></br>
+  Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
+  ```
+  BuildWorker.bat <YourProject>SimulatedPlayer Linux <Configuration> <YourProject>.uproject
```
Comment (improbable-valentyn) [inline SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-cloud.md]: And here as well, engine plugin and project plugin alternatives.

File: SpatialGDK/Documentation/content/cloud-deployment-workflow.md (hunk 1 above)
Comment (ernopp) [reply SpatialGDK/Documentation/content/cloud-deployment-workflow.md]: For each of the 3 docs pages in this PR, we explain the full filepath of Buildworker.bat above (when we explain how to build the client and server worker assemblies) 

By this point people will know where their buildworker.bat file is so I feel we don't need to re-explain it

Comment (ernopp) [reply SpatialGDK/Documentation/content/cloud-deployment-workflow.md]: Actually you're right for consistency let's keep listing the engine and project plugin filepaths - updated

**Pull request merged**